draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -211,6 +224,14 @@ when, and only when, they appear in all capitals, as shown here.
 
 > **Editor's note:** the list of terms below will almost certainly grow in size as the specification matures.
 
+> **Note to the Reader:** the meaning of the terms "congestion control" and "rate adaptation" in the IETF community have evolved over the decades since "slow start" and "congestion avoidance" were added as mandatory to implement in TCP, in {{Section 4.2.2.15 of ?RFC1122}}. At that time, "congestion control" usually referred to "achieving network stability" ({{VJMK88}}), by protecting the network from senders who continue to transmit packets that exceed the ability of the network to carry them, even after packet loss occurs (called "congestion collapse").
+
+> "Rate adaptation" more commonly referred to strategies intended to guide senders on when to send "the next packet", so that one-way delays along the network path remain minimal.
+
+> As more and more general-purpose "congestion control" algorithms focused on avoiding "bufferbloat", as described in {{nested-CC}}, the difference between "congestion control" and "rate adaptation" has blurred in IETF community discussions.
```

Comment:
In #128 I removed the {{nested-CC}} section and merged it with the one before because they became very similar. I also removed the note about bufferbloat. Maybe we can directly include that here and remove the reference to the other section?

Reply:
Hi, @mengelbart - that sounds good, but I'm not sure whether you wanted to fix this, or have me fix it. Since we have a conference call in 9 minutes, let's fix it exactly once!